WcaOnRails/config/locales/kaminari.en.yml

```diff
@@ -0,0 +1,19 @@
+# Sample localization file for Kaminari. You can override these values in your app's locales file if you want.
+
+en:
+  views:
+    pagination:
+      first: "&laquo; First"
+      last: "Last &raquo;"
+      previous: "&lsaquo; Prev"
+      next: "Next &rsaquo;"
+      truncate: "&hellip;"
+  helpers:
+    page_entries_info:
+      one_page:
+        display_entries:
+          zero: "No %{entry_name} found"
+          one: "Displaying <b>1</b> %{entry_name}"
```

Comment:
Ahah, absolutely not!

I did not notice that; I took 'kaminari.*.yml' from kaminari's provided locales, I would rather not modify them unless we need to.
In this particular case our app doesn't use these entries, maybe they were meant to be used by kaminari which takes care of the html-ization?